CodeCoverage.runsettings

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Should we move this file into the `test` directory? This would keep the root directory a little bit tidier